### ISSUE TITLE
Redline Racer - Wrong message is displayed

### ISSUE THREAD

**Issue by kihato** [flyinghead/flycast#1068]
Platform / OS / Hardware: 
Windows11 / i7-10700F / GTX 1660 SUPER

Github hash: 5415442

![Redlineraser](https://github.com/flyinghead/flycast/assets/14366233/bf8838d4-6862-4f3a-b3f6-8107fe9538da)

Not enough free blocks.
Saving requires 12 blocks of free space.
In this state, you can not save even if you start the game.

However, this message is displayed even if there is free space or there is a save file.

**Comment by flyinghead** [comment]
The game looks for vmu save files without waiting for the initial vmu scan to finish, which causes this error. Interestingly the US and EU versions (Suzuki ALSTARE Extreme Racing) don't have have this issue so it makes me believe it's a game bug.
I was able to fix the problem by slowing down the GD-ROM speed, which might be the reason why it doesn't happen on real hw. Another work-around is to disconnect then reconnect the controller or vmu after the game has started. I was able to save and load my progress by doing that although I'm not sure it works 100%.
I would recommend to run the US or EU edition for now since I don't have any easy code fix.
